sites/mock/package.json

```diff
@@ -8,9 +8,12 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@brave/react-virtualized-auto-sizer": "^1.0.4",
```

Comment:
Just wondering if we need these dependencies in `sites/mock`. IIUC, `@brave/swap-interface` should take care of it.

Reply:
Removed!